tests/pds/pds_storagesize_increase_test.go

```diff
@@ -48,11 +49,23 @@ var _ = Describe("{ResizeStorageAndRestoreWithVariousFSandRepl}", func() {
 			newResourceTemplateID    string
 			newStorageTemplateID     string
 			updatedPvcSize           uint64
+			beforeResizePodAge       float64
 		)
+		credName := targetName + pdsbkp.RandString(8)
+		bkpClient, err = pdsbkp.InitializePdsBackup()
+		log.FailOnError(err, "Failed to initialize backup for pds.")
+		stepLog := "Create AWS S3 Backup target."
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			bkpTarget, err := bkpClient.CreateAwsS3BackupCredsAndTarget(tenantID, fmt.Sprintf("%v-aws", credName), deploymentTargetID)
```

Comment:
This is taken care in before suite, kindly remove creation and deletion of bucket creation